### PR TITLE
move project_partner_name from project_partner_mandatory

### PR DESCRIPTION
**Pull request by noahzaozao**

### CODE REVIEW

File: project_partner_name/README.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+=========================
+Project Partner Name
+=========================
+
+This module extends the functionality of Project:
+
+* Makes "Customer" mandatory field.
+* Modifies the name_get adding the customer reference to the project name.
+* Modifies the name_search to search also by customer reference linked to project.
```
Comment (elicoidal) [inline project_partner_name/README.rst]: does it solve the 2 pending bugs:
- name_get of analytic account should be added as well
- when in task / project field: the name of the first customer is repeated in all lines

File: project_partner_name/README.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (project)
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/elico_odoo/issues>`_.
```
Comment (elicoidal) [inline project_partner_name/README.rst]: update to latest OCA version

File: project_partner_name/README.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/Elico-Corp/elico_odoo/issues/new?body=module:%20read_customer_reference%0Aversion:%20{8.0.1.0.1}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Victor M. Martin <victor.martin@elico-corp.com>
```
Comment (elicoidal) [inline project_partner_name/README.rst:36]: add Noah + Eric

File: project_partner_name/README.rst [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+Maintainer
+----------
+
+.. image:: https://www.elico-corp.com/logo.png
+:alt: Elico Corp
+:target: https://www.elico-corp.com
+
+This module is maintained by Elico Corporation.
+
+Elico Corporation offers consulting services to implement open source management software in SMEs, with a strong involvement in quality of service.
```
Comment (elicoidal) [inline project_partner_name/README.rst]: modify this footer with <80 char lines (and please inform everybody about the new footer)

**Review by elicoidal — CHANGES_REQUESTED** [review]
Check the current pending bugs

File: project_partner_name/README.rst (hunk 1 above)
Comment (noahzaozao) [reply project_partner_name/README.rst]: @elicoidal this PR is only move the module and rename it from elico_odoo , after it merged , Frank will work on these changes and create another PR for the modification.

File: project_partner_name/README.rst (hunk 3 above)
Comment (noahzaozao) [reply project_partner_name/README.rst:36]: @elicoidal added

**Comment by noahzaozao** [comment]
@seb-elico @elicoidal Please review this pr thanks

File: project_partner_name/README.rst [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+=========================
+Project Partner Name
+=========================
+
+This module extends the functionality of Project:
+
+* Makes "Customer" mandatory field.
+* Modifies the name_get adding the customer reference to the project name.
+* Modifies the name_search to search also by customer reference linked to
+project.
```
Comment (elicoidal) [inline project_partner_name/README.rst:14]: add 2 spaces at the beginning of the line

```
* Modifies the name_search to search also by customer reference linked to
  project
```

**Review by elicoidal — CHANGES_REQUESTED** [review]
LGTM (small nitpick)

**Comment by noahzaozao** [comment]
@seb-elico @elicoidal Please review this pr thanks

**Comment by xuan139** [comment]
LGTM

File: project_partner_name/README.rst [hunk 6]
```diff
@@ -0,0 +1,54 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Project Partner Name
+=========================
+
+This module extends the functionality of Project:
+  * Makes "Customer" mandatory field.
```
Comment (elicoidal) [inline project_partner_name/README.rst:10]: add a blank  line before list

**Pull request merged**